src/popup/components/Modals/ConfirmTransactionSign.vue

```diff
@@ -548,25 +514,26 @@ export default defineComponent({
       getTxKeyLabel,
       getTxAssetSymbol,
       isAeppChatSuperhero,
+      isAex9,
       isDex,
       isDexAllowance,
+      isDexMaxSpent,
+      isDexMinReceived,
+      isDexSwap,
       isHash,
-      isSwap,
-      isTransactionAex9,
       loading,
       nameAeFee,
       popupProps,
       showAdvanced,
```

Comment:
This is legacy variable, it is not used.

Reply:
Sounds reasonable. Removed...